src/common/options.cc

```diff
@@ -6963,6 +6963,16 @@ std::vector<Option> get_rgw_options() {
       "rgw_crypt_vault_auth",
       "rgw_crypt_vault_addr"}),
 
+    Option("rgw_crypt_vault_role", Option::TYPE_STR, Option::LEVEL_ADVANCED)
```

Comment:
there are already a lot of config options for vault. if this one is specific to service account, would it make sense to call this `rgw_crypt_vault_service_account_role` to clarify?